### PR TITLE
Combine variable and strings for translation: Modify jump block

### PR DESCRIPTION
**Pull request by nkiruka**
Added new strings to turtle/en_us.json and updated jump function to use interpolateMsg 

Screen Shot 2019-09-03 at 1 17 25 PM

### CODE REVIEW

File: apps/src/turtle/blocks.js [hunk 1]
```diff
@@ -717,17 +717,22 @@ exports.install = function(blockly, blockInstallOptions) {
...
-      this.appendDummyInput().appendTitle(msg.dots());
+      this.interpolateMsg(
+        msg.jumpToDirectionByPixels(),
+        () => {
+          this.appendDummyInput().appendTitle(
+            new blockly.FieldDropdown(blockly.Blocks.jump.DIRECTIONS),
+            'DIR'
+          );
+        },
+        ['BY', 'Number', blockly.ALIGN_RIGHT],
```
Comment (joshlory) [inline apps/src/turtle/blocks.js]: I think the extra socket is coming from this line.

File: apps/i18n/turtle/en_us.json [hunk 2]
```diff
@@ -39,12 +40,14 @@
...
   "drawSticker": "draw sticker",
   "drawStickerWithSize": "draw sticker with size",
+  "forward": "forward",
   "heightParameter": "height",
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpTo": "jump to",
   "jumpBackward": "jump backward by",
   "jumpForward": "jump forward by",
+  "jumpToDirectionByPixels": "jump %1 by %2 pixels",
```
Comment (Hamms) [inline apps/i18n/turtle/en_us.json]: why "ToDirection"?

File: apps/i18n/turtle/en_us.json [hunk 3]
```diff
@@ -39,12 +40,14 @@
   "drawLowerWave": "draw lower wave",
   "drawSticker": "draw sticker",
   "drawStickerWithSize": "draw sticker with size",
+  "forward": "forward",
   "heightParameter": "height",
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpTo": "jump to",
   "jumpBackward": "jump backward by",
   "jumpForward": "jump forward by",
```
Comment (Hamms) [inline apps/i18n/turtle/en_us.json:48]: Can these old strings be removed?

File: apps/i18n/turtle/en_us.json [hunk 4]
```diff
@@ -39,12 +40,12 @@
...
   "drawSticker": "draw sticker",
   "drawStickerWithSize": "draw sticker with size",
+  "forward": "forward",
   "heightParameter": "height",
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpToOverDown": "jump to %1 over %2 down",
-  "jumpBackward": "jump backward by",
-  "jumpForward": "jump forward by",
+  "jumpToByPixels": "jump %1 by %2 pixels",
```
Comment (Hamms) [inline apps/i18n/turtle/en_us.json]: Why "To"?

**Review by Hamms — APPROVED** [review]

**Pull request merged**